@xen-orchestra/fs/src/fs.spec.js

```diff
@@ -116,6 +116,26 @@ handlers.forEach(url => {
       })
     })
 
+    describe('#getInfo()', () => {
+      let info
+      beforeAll(async () => {
+        info = await handler.getInfo()
+      })
+
+      it('should return an object with info', async () => {
+        expect(typeof info).toBe('object')
+      })
+
+      it('should return correct type of attribute', async () => {
+        if (info.available) {
```

Comment:
Explicit test please, for instance here you could do `'available' in info` or `info.available !== undefined`.